README.md

```diff
@@ -1,2 +1,21 @@
 # orb_slam2_catkin
-Catkin wrapper for orbslam2 package. Tested on 16.04 with Kinetic and 14.04 with Indigo.
+
+## Overview
+
+This package is a catkinized version of the original library [ORB_SLAM2](https://github.com/raulmur/ORB_SLAM2)
+
+**Keywords:** CATKIN, ORB_SLAM2, SLAM
+
+### License
+
+The source code is released under a [TODO @alexmillane insert license](put_link_here).
+
+**Author(s): Alex Millane
+Maintainer: Alex Millane
```

Comment:
Oh and add yourself to these lists!.. If you want.

Reply:
nope!

Reply:
Also, can you please add a license to his before merging in?